client/packages/common/src/ui/components/buttons/standard/TextInputButton.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+  Button as MuiButton,
+  ButtonProps as MuiButtonProps,
+  styled,
+} from '@mui/material';
+
+export const StyledTextInputButton = styled(MuiButton)(({
+  theme,
+  disabled,
+}) => {
+  return {
+    color: theme.palette.gray.dark,
+
+    backgroundColor: disabled
+      ? theme.palette.background.toolbar
+      : theme.palette.background.menu,
+    height: '36px',
+    justifyContent: 'space-between',
+    '& .MuiButton-endIcon': {
+      marginRight: '2px',
+    },
+    fontWeight: 'unset',
+    fontSize: 'unset',
+    width: '100%',
+    textTransform: 'none',
+  };
+});
+
+export const TextInputButton = ({ onClick, ...rest }: MuiButtonProps) => {
+  return (
+    <StyledTextInputButton
```

Comment:
Thought new kind of button would be best here, semantically it should be a button, we want the whole thing clickable, to open the popup